src/Datadog.Trace/TracingProcessManager.cs

```diff
@@ -146,25 +183,51 @@ private static void InitializePortManagerClaimFiles(string traceAgentDirectory)
                 Directory.CreateDirectory(portManagerDirectory);
             }
 
-            var currentProcess = Process.GetCurrentProcess();
-            _processName = currentProcess.ProcessName.ToLowerInvariant();
-            _processId = currentProcess.Id;
-            var fileClaim = Path.Combine(portManagerDirectory, _processId.ToString());
+            var fileClaim =
+                Path.Combine(
+                    portManagerDirectory,
+                    $"{DomainMetadata.ProcessId}_{DomainMetadata.AppDomainId}");
```

Comment:
We should specify the invariant culture in this conversion. This would work in 4.6 and above:
```csharp
FormattableString.Invariant($"{DomainMetadata.ProcessId}_{DomainMetadata.AppDomainId}");
```

Since we need to support 4.5, we may need to remove the interpolated string and go old school:
```suggestion
                    string.Format(CultureInfo.InvariantCulture, "{0}_{1}", DomainMetadata.ProcessId, DomainMetadata.AppDomainId);
```